requirements.txt

```diff
@@ -1,10 +1,12 @@
 # Prepare for Documentation
 lxml
 pyyaml
+click
 
 # Documentation building
 sphinx>=5
 sphinx-tabs
+sphinx-toolbox
```

Comment:
Missing from `pyproject.toml`?

Reply:
sphinx-toolbox was already included in pyproject.toml.